src/Error.cpp

```diff
@@ -1,11 +1,32 @@
 #include "Error.h"
 
+#include <signal.h>
+
+#ifdef _WIN32
+#include <windows.h>
+#endif
+
 namespace Halide {
 
 namespace {
 
 CompileTimeErrorReporter* custom_error_reporter = nullptr;
 
+void error_abort() {
+#if defined(_MSC_VER) && defined(_DEBUG)
+    if (!IsDebuggerPresent()) {
```

Comment:
I don't think this really solves the problem. When I worked on windows a lot, my main use of this dialog was when I *wasn't* running in the debugger, and then hitting retry would start the debugger and let you debug the process after the fact (without starting from the debugger).

Reply:
Sigh. Yeap. You're right.